{
  "uid" : "71de5a63aaeb9842",
  "name" : "Проверка текста кнопки 'Войти'",
  "fullName" : "tests.test_header.test_header.TestHeader#test_get_text_header_auth_login",
  "historyId" : "9f8cd45a87327236c649203f54924218",
  "time" : {
    "start" : 1714443569313,
    "stop" : 1714443569313,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x55e8f0b34e33 <unknown>\n#1 0x55e8f0824567 <unknown>\n#2 0x55e8f081c879 <unknown>\n#3 0x55e8f080c852 <unknown>\n#4 0x55e8f080de44 <unknown>\n#5 0x55e8f080cbbe <unknown>\n#6 0x55e8f080c2e0 <unknown>\n#7 0x55e8f080c181 <unknown>\n#8 0x55e8f080a0ec <unknown>\n#9 0x55e8f080a60a <unknown>\n#10 0x55e8f0826eef <unknown>\n#11 0x55e8f08b0695 <unknown>\n#12 0x55e8f0891362 <unknown>\n#13 0x55e8f08afae2 <unknown>\n#14 0x55e8f0891103 <unknown>\n#15 0x55e8f08621ba <unknown>\n#16 0x55e8f0862b8e <unknown>\n#17 0x55e8f0af959b <unknown>\n#18 0x55e8f0afd34b <unknown>\n#19 0x55e8f0ae5681 <unknown>\n#20 0x55e8f0afdeb2 <unknown>\n#21 0x55e8f0aca6af <unknown>\n#22 0x55e8f0b24428 <unknown>\n#23 0x55e8f0b24600 <unknown>\n#24 0x55e8f0b33f84 <unknown>\n#25 0x7f108d294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c3f7030c58d649413c225a69a7941809\")>\nurl = 'https://pulsewave.ru/'\n\n    @pytest.fixture()\n    def header_open(driver, url):\n        page = HeaderPage(driver)\n>       driver.get(url)\n\ntests/test_header/conftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:356: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f17c7dc74d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55e8f0b24600 \\\\u003Cunknown>\\\\n#24 0x55e8f0b33f84 \\\\u003Cunknown>\\\\n#25 0x7f108d294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome-headless-shell=124.0.6367.60)\nE       Stacktrace:\nE       #0 0x55e8f0b34e33 <unknown>\nE       #1 0x55e8f0824567 <unknown>\nE       #2 0x55e8f081c879 <unknown>\nE       #3 0x55e8f080c852 <unknown>\nE       #4 0x55e8f080de44 <unknown>\nE       #5 0x55e8f080cbbe <unknown>\nE       #6 0x55e8f080c2e0 <unknown>\nE       #7 0x55e8f080c181 <unknown>\nE       #8 0x55e8f080a0ec <unknown>\nE       #9 0x55e8f080a60a <unknown>\nE       #10 0x55e8f0826eef <unknown>\nE       #11 0x55e8f08b0695 <unknown>\nE       #12 0x55e8f0891362 <unknown>\nE       #13 0x55e8f08afae2 <unknown>\nE       #14 0x55e8f0891103 <unknown>\nE       #15 0x55e8f08621ba <unknown>\nE       #16 0x55e8f0862b8e <unknown>\nE       #17 0x55e8f0af959b <unknown>\nE       #18 0x55e8f0afd34b <unknown>\nE       #19 0x55e8f0ae5681 <unknown>\nE       #20 0x55e8f0afdeb2 <unknown>\nE       #21 0x55e8f0aca6af <unknown>\nE       #22 0x55e8f0b24428 <unknown>\nE       #23 0x55e8f0b24600 <unknown>\nE       #24 0x55e8f0b33f84 <unknown>\nE       #25 0x7f108d294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714443483373,
      "stop" : 1714443483373,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714443483355,
      "stop" : 1714443483372,
      "duration" : 17
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1714443569314,
      "stop" : 1714443569511,
      "duration" : 197
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "header_open",
    "time" : {
      "start" : 1714443569511,
      "stop" : 1714443569729,
      "duration" : 218
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x55e8f0b34e33 <unknown>\n#1 0x55e8f0824567 <unknown>\n#2 0x55e8f081c879 <unknown>\n#3 0x55e8f080c852 <unknown>\n#4 0x55e8f080de44 <unknown>\n#5 0x55e8f080cbbe <unknown>\n#6 0x55e8f080c2e0 <unknown>\n#7 0x55e8f080c181 <unknown>\n#8 0x55e8f080a0ec <unknown>\n#9 0x55e8f080a60a <unknown>\n#10 0x55e8f0826eef <unknown>\n#11 0x55e8f08b0695 <unknown>\n#12 0x55e8f0891362 <unknown>\n#13 0x55e8f08afae2 <unknown>\n#14 0x55e8f0891103 <unknown>\n#15 0x55e8f08621ba <unknown>\n#16 0x55e8f0862b8e <unknown>\n#17 0x55e8f0af959b <unknown>\n#18 0x55e8f0afd34b <unknown>\n#19 0x55e8f0ae5681 <unknown>\n#20 0x55e8f0afdeb2 <unknown>\n#21 0x55e8f0aca6af <unknown>\n#22 0x55e8f0b24428 <unknown>\n#23 0x55e8f0b24600 <unknown>\n#24 0x55e8f0b33f84 <unknown>\n#25 0x7f108d294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_header/conftest.py\", line 8, in header_open\n    driver.get(url)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714443569761,
      "stop" : 1714443569818,
      "duration" : 57
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование Хедера"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_header"
  }, {
    "name" : "suite",
    "value" : "test_header"
  }, {
    "name" : "subSuite",
    "value" : "TestHeader"
  }, {
    "name" : "host",
    "value" : "fv-az1385-609"
  }, {
    "name" : "thread",
    "value" : "2312-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_header.test_header"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  }, {
    "name" : "url",
    "value" : "'https://pulsewave.ru/'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 14
      },
      "items" : [ {
        "uid" : "4e0a79340114614f",
        "reportUrl" : "https://Nat754.github.io/PulseWave/9//#testresult/4e0a79340114614f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x55757db03e33 <unknown>\n#1 0x55757d7f3567 <unknown>\n#2 0x55757d7eb879 <unknown>\n#3 0x55757d7db852 <unknown>\n#4 0x55757d7dce44 <unknown>\n#5 0x55757d7dbbbe <unknown>\n#6 0x55757d7db2e0 <unknown>\n#7 0x55757d7db181 <unknown>\n#8 0x55757d7d90ec <unknown>\n#9 0x55757d7d960a <unknown>\n#10 0x55757d7f5eef <unknown>\n#11 0x55757d87f695 <unknown>\n#12 0x55757d860362 <unknown>\n#13 0x55757d87eae2 <unknown>\n#14 0x55757d860103 <unknown>\n#15 0x55757d8311ba <unknown>\n#16 0x55757d831b8e <unknown>\n#17 0x55757dac859b <unknown>\n#18 0x55757dacc34b <unknown>\n#19 0x55757dab4681 <unknown>\n#20 0x55757dacceb2 <unknown>\n#21 0x55757da996af <unknown>\n#22 0x55757daf3428 <unknown>\n#23 0x55757daf3600 <unknown>\n#24 0x55757db02f84 <unknown>\n#25 0x7f23bec94ac3 <unknown>",
        "time" : {
          "start" : 1714357226465,
          "stop" : 1714357226465,
          "duration" : 0
        }
      }, {
        "uid" : "38526ecc16fa091a",
        "reportUrl" : "https://Nat754.github.io/PulseWave/8//#testresult/38526ecc16fa091a",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5620b9e62e33 <unknown>\n#1 0x5620b9b52567 <unknown>\n#2 0x5620b9b9d27d <unknown>\n#3 0x5620b9b9d331 <unknown>\n#4 0x5620b9be0624 <unknown>\n#5 0x5620b9bbf38d <unknown>\n#6 0x5620b9bddae2 <unknown>\n#7 0x5620b9bbf103 <unknown>\n#8 0x5620b9b901ba <unknown>\n#9 0x5620b9b90b8e <unknown>\n#10 0x5620b9e2759b <unknown>\n#11 0x5620b9e2b34b <unknown>\n#12 0x5620b9e13681 <unknown>\n#13 0x5620b9e2beb2 <unknown>\n#14 0x5620b9df86af <unknown>\n#15 0x5620b9e52428 <unknown>\n#16 0x5620b9e52600 <unknown>\n#17 0x5620b9e61f84 <unknown>\n#18 0x7ff1e7494ac3 <unknown>",
        "time" : {
          "start" : 1714272170603,
          "stop" : 1714272170603,
          "duration" : 0
        }
      }, {
        "uid" : "ecd02b42cd09f1e3",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/ecd02b42cd09f1e3",
        "status" : "passed",
        "time" : {
          "start" : 1714184897194,
          "stop" : 1714184897234,
          "duration" : 40
        }
      }, {
        "uid" : "581837fb8b8e22aa",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/581837fb8b8e22aa",
        "status" : "passed",
        "time" : {
          "start" : 1714148381549,
          "stop" : 1714148381592,
          "duration" : 43
        }
      }, {
        "uid" : "ee8a636f275085b0",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/ee8a636f275085b0",
        "status" : "passed",
        "time" : {
          "start" : 1714098429607,
          "stop" : 1714098429646,
          "duration" : 39
        }
      }, {
        "uid" : "55ba8e3083439864",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/55ba8e3083439864",
        "status" : "passed",
        "time" : {
          "start" : 1714014323106,
          "stop" : 1714014323144,
          "duration" : 38
        }
      }, {
        "uid" : "c3c2d9a13ef55cf6",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/c3c2d9a13ef55cf6",
        "status" : "passed",
        "time" : {
          "start" : 1713945813813,
          "stop" : 1713945813850,
          "duration" : 37
        }
      }, {
        "uid" : "3d0134dd4c9001f4",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/3d0134dd4c9001f4",
        "status" : "passed",
        "time" : {
          "start" : 1713925606785,
          "stop" : 1713925606824,
          "duration" : 39
        }
      }, {
        "uid" : "bca8116842479e2d",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/bca8116842479e2d",
        "status" : "passed",
        "time" : {
          "start" : 1713901451009,
          "stop" : 1713901451051,
          "duration" : 42
        }
      }, {
        "uid" : "a5faabd99ecdea11",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/a5faabd99ecdea11",
        "status" : "passed",
        "time" : {
          "start" : 1713885632162,
          "stop" : 1713885632201,
          "duration" : 39
        }
      }, {
        "uid" : "b2327277fd0241f4",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/b2327277fd0241f4",
        "status" : "passed",
        "time" : {
          "start" : 1713883534909,
          "stop" : 1713883534947,
          "duration" : 38
        }
      }, {
        "uid" : "1dbb9bb1e664710b",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/1dbb9bb1e664710b",
        "status" : "passed",
        "time" : {
          "start" : 1713882047574,
          "stop" : 1713882047614,
          "duration" : 40
        }
      }, {
        "uid" : "639f78d1168291d9",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/639f78d1168291d9",
        "status" : "passed",
        "time" : {
          "start" : 1713879222764,
          "stop" : 1713879222803,
          "duration" : 39
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "71de5a63aaeb9842.json",
  "parameterValues" : [ "'chrome'", "'https://pulsewave.ru/'" ]
}